scripts/init.sql

```diff
@@ -1,289 +1,228 @@
-ADD
-    CONSTRAINT wishcards_child_id_fkey FOREIGN KEY (child_id) REFERENCES children (id);
-
-ALTER TABLE wishcards
-ADD
-    CONSTRAINT wishcards_agency_id_fkey FOREIGN KEY (agency_id) REFERENCES agencies (id) ON DELETE Cascade;
-
-ALTER TABLE wishcards
-ADD
-    CONSTRAINT wishcards_created_by_fkey FOREIGN KEY (created_by) REFERENCES users (id);
-
-ALTER TABLE wishcards
-ADD
-    CONSTRAINT wishcards_item_id_fkey FOREIGN KEY (item_id) REFERENCES items (id);
-
-ALTER TABLE messages
-ADD
-    CONSTRAINT messages_wishcard_id_fkey FOREIGN KEY (wishcard_id) REFERENCES wishcards (id);
-
-ALTER TABLE wishcards
-ADD
-    CONSTRAINT wishcards_image_id_fkey FOREIGN KEY (image_id) REFERENCES images (id);
-
-ALTER TABLE children
-ADD
-    CONSTRAINT children_image_id_fkey FOREIGN KEY (image_id) REFERENCES images (id);
-
-ALTER TABLE items
-ADD
-    CONSTRAINT items_image_id_fkey FOREIGN KEY (image_id) REFERENCES images (id);
-
-ALTER TABLE agencies
-ADD
-    CONSTRAINT agencies_image_id_fkey FOREIGN KEY (image_id) REFERENCES images (id);
-
-ALTER TABLE community_posts
-ADD
-    CONSTRAINT community_posts_image_id_fkey FOREIGN KEY (image_id) REFERENCES images (id);
-
-ALTER TABLE users
-ADD
-    CONSTRAINT users_image_id_fkey FOREIGN KEY (image_id) REFERENCES images (id);
-
-ALTER TABLE orders
-ADD
-    CONSTRAINT orders_donor_id_fkey FOREIGN KEY (donor_id) REFERENCES users (id);
-
-ALTER TABLE orders
-ADD
-    CONSTRAINT orders_child_id_fkey FOREIGN KEY (child_id) REFERENCES children (id);
-
-ALTER TABLE orders
-ADD
-    CONSTRAINT orders_item_id_fkey FOREIGN KEY (item_id) REFERENCES items (id);
-
-ALTER TABLE orders
-ADD
-    CONSTRAINT orders_agency_id_fkey FOREIGN KEY (agency_id) REFERENCES agencies (id);
-
-ALTER TABLE wishcards
-ADD
-    CONSTRAINT wishcards_order_id_fkey FOREIGN KEY (order_id) REFERENCES orders (id);
-
--- functions
-CREATE OR REPLACE FUNCTION public.TRIGGER_SET_UPDATED_DATE()
-    RETURNS TRIGGER
-    LANGUAGE PLPGSQL
+CREATE TABLE "agencies" (
+    "id" uuid PRIMARY KEY NOT NULL DEFAULT (gen_random_uuid()),
+    "name" varchar(255) NOT NULL,
+    "website" varchar(255) NOT NULL,
+    "phone" varchar(100) NOT NULL,
+    "bio" varchar(500),
+    "address_line_1" varchar(255) NOT NULL,
+    "address_line_2" varchar(255),
+    "city" varchar(100) NOT NULL,
+    "state" varchar(100) NOT NULL,
+    "country" varchar(100) NOT NULL,
+    "zip_code" varchar(50) NOT NULL,
+    "verified" boolean NOT NULL DEFAULT false,
+    "employer_identification_number" varchar(255) UNIQUE,
+    "account_manager_id" uuid NOT NULL,
+    "image_id" uuid NOT NULL,
+    "created_at" timestamp NOT NULL DEFAULT (CURRENT_TIMESTAMP),
+    "updated_at" timestamp
+);
+
+CREATE TABLE "children" (
+    "id" uuid PRIMARY KEY NOT NULL DEFAULT (gen_random_uuid()),
+    "first_name" varchar(255) NOT NULL,
+    "last_name" varchar(255) NOT NULL,
+    "birth_year" integer NOT NULL,
+    "interest" varchar(255) NOT NULL,
+    "story" varchar(255) NOT NULL,
```

Comment:
Could we change the story varchar to 2000 to cover edge cases with long stories?

Reply:
In those cases maybe its better to leave it as text type without a length limit?